ROSCO/src/DISCON.F90

```diff
@@ -87,12 +97,12 @@ SUBROUTINE DISCON(avrSWAP, aviFAIL, accINFILE, avcOUTNAME, avcMSG) BIND (C, NAME
     IF (CntrPar%Flp_Mode > 0) THEN
         CALL FlapControl(avrSWAP, CntrPar, LocalVar, objInst)
     END IF
-    
-    IF (CntrPar%LoggingLevel > 0) THEN
-        CALL Debug(LocalVar, CntrPar, DebugVar, avrSWAP, RootName, SIZE(avcOUTNAME))
-    END IF
+
+
 END IF
 
+CALL Debug(LocalVar, CntrPar, DebugVar, avrSWAP, RootName, SIZE(avcOUTNAME))

```

Comment:
Let's get this back in an if statement like before. It can cause some processing delays for real-time applications.